From 7739c21302e62c8a45062ccbbee2e27b8849a9fa Mon Sep 17 00:00:00 2001
From: Kexy Biscuit <kexybiscuit@aosc.io>
Date: Mon, 12 Aug 2024 08:06:38 +0800
Subject: [PATCH 044/103] PCI: Add Phytium vendor ID

Update pci_ids.h with the vendor ID for Phytium.

Signed-off-by: Chen Baozi <chenbaozi@phytium.com.cn>
Change-Id: Ie0ea6afa71a8ba2f66f8426c1db063cd734d985c
Ref: https://gitee.com/phytium_embedded/phytium-linux-kernel/commit/b570810d41c9cb46f05c31b51d156c6b4441486a
---
 include/linux/pci_ids.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/pci_ids.h b/include/linux/pci_ids.h
index 677aea20d3e11..5b923cb5698c5 100644
--- a/include/linux/pci_ids.h
+++ b/include/linux/pci_ids.h
@@ -3224,4 +3224,6 @@
 
 #define PCI_VENDOR_ID_NCUBE		0x10ff
 
+#define PCI_VENDOR_ID_PHYTIUM		0x1db7
+
 #endif /* _LINUX_PCI_IDS_H */
-- 
2.46.0

